src/buttons/_base.scss

```diff
@@ -159,38 +159,34 @@
 
 [data-button="play"] {
   position: relative;
-  display: inline-block;
-  min-height: 56px;
-  border-radius: 35px;
+  min-height: 3.5rem;
+  border-radius: 2.188rem;
```

Comment:
@misscs I think we talked about 2 or 3 places. I like to do 3 for precision, but not sure if it makes much difference.